docs/development.rst

```diff
@@ -44,40 +44,42 @@ independent parts which can be reviewed and merged individually.
 Automated Testing
 =================
 
-All pull requests and merges to 'master' branch are tested in `Travis <https://travis-ci.org/>`_
-based on our `.travis.yml file <https://github.com/pypa/pip/blob/master/.travis.yml>`_.
+All pull requests and merges to 'master' branch are tested using `Travis CI`_
+based on `our .travis.yml file`_.
 
-Usually, a link to your specific travis build appears in pull requests, but if not,
-you can find it on our `travis pull requests page <https://travis-ci.org/pypa/pip/pull_requests>`_
+Usually, a link to your specific Travis CI build appears in pull requests, but
+if not, you can find it on `our Travis CI Pull Requests page`_
 
-The only way to trigger Travis to run again for a pull request, is to submit another change to the pull branch.
+The only way to trigger Travis to run again for a pull request, is to submit
+another change to the pull branch.
 
-We also have Jenkins CI that runs regularly for certain python versions on windows and centos.
+We also have Jenkins CI that runs regularly for certain python versions on
+Windows and CentOS.
 
 Running tests
 =============
 
 OS Requirements: subversion, bazaar, git, and mercurial.
 
-Python Requirements: tox or pytest, virtualenv, scripttest, and mock
+Python Requirements: tox or pytest, virtualenv, scripttest, PyYAML, and mock
```

Comment:
There's more than those - maybe better just refer to `dev-requirements.txt`.

Reply:
These are the minimum requirements. The others are basically add-ons for making CI better.

That said, linking to `dev-requirements.txt` does sound better to me. I guess a link to it on Github's preview of the file on master should be fine.